riocli/deployment/util.py

```diff
@@ -77,10 +75,11 @@ def get_deployment_guid(client: Client, name: str) -> str:
 def get_deployment_name(client: Client, guid: str) -> str:
```

Comment:
question: Is this function still used?

Reply:
Yes, in the name to guid decorator. It uses the v2 client.